src/main/java/dto/TransferObject.java

```diff
@@ -0,0 +1,34 @@
+package dto;
+
+public class TransferObject {
```

Comment:
```suggestion
public class Transaction {
```
let's use a more informative naming here, TransferObject is too generic